dkist/net/attr_walker.py

```diff
@@ -143,6 +147,20 @@ def _(wlk, attr, params):
     return params.update({'embargoEndDateMin': attr.start.isot,
                           'embargoEndDateMax': attr.end.isot})
 
+@walker.add_applier(SpectralSampling)
+def _(wlk, attr, params):
+    return params.update({'averageDatasetSpectralSamplingMin': attr.min.to_value(equivalencies=float),
+                          'averageDatasetSpectralSamplingMax': attr.max.to_value(equivalencies=float)})
```

Comment:
You need to give the target unit here.
```suggestion
    return params.update({'averageDatasetSpectralSamplingMin': attr.min.to_value(u.nm, equivalencies=u.spectral()),
                          'averageDatasetSpectralSamplingMax': attr.max.to_value(u.nm, equivalencies=u.spectral())})

```